src/components/StatusBadge/StatusBadge.mdx

```diff
@@ -51,3 +53,15 @@ Used in instances where the associated message is the lower barrier. Use
 `"low"` for this level which is the default.
 
 <Canvas of={StatusBadgeStories.LowerBarrier} />
+
+### Icons
+
+You may render an icon within the `StatusBadge` by passing it as a child to the component.
+If you are using an icon, you should still pass text to the badge so that the user can
+understand its meaning.
+
```

Comment:
Can you add a visual code snippet here to show the pattern for adding an icon and text inside of the `StatusBadge`?